src/lib/containers/download_list_v2/DownloadListTable.tsx

```diff
@@ -158,6 +159,10 @@ export default function DownloadListTable() {
                   Name
                   <span>{showInteractiveSortIcon('fileName')}</span>
                 </th>
+                <th>
```

Comment:
Move the file size column over to the left - my understanding is that design generally wants to move more important items to the left in the table so that they are initially seen